lib/response.js

```diff
@@ -22,7 +22,7 @@ module.exports = (function () {
             this.handler.response = buildSpeechletResponse({
                 sessionAttributes: this.attributes,
                 output: getSSMLResponse(speechOutput),
-                reprompt: getSSMLResponse(repromptSpeech),
+                reprompt: getSSMLResponse(repromptSpeech || speechOutput),
```

Comment:
I think the current system is also good as it encourages developers to have different reprompts to the original output, which is best practice for VUIs.